pkg/database/service.go

```diff
@@ -193,7 +150,7 @@ func (s service) Download(id uint, dst io.Writer, cb func(contentLength int64))
 	}
 
 	if d.Url == "" {
-		return apperror.NewBadRequest(fmt.Sprintf("database with %d doesn't reference any url", id))
+		return errdef.NewBadRequest("database with %d doesn't reference any url", id)
```

Comment:
```suggestion
		return errdef.NewBadRequest("database with id %d doesn't reference any url", id)
```

might not be clear what `database with %d` means